threat_intel/virustotal.py

```diff
@@ -66,6 +117,45 @@ def get_domain_reports(self, domains):
         return all_responses
 
     @MultiRequest.error_handling
+    def get_url_distribution(self, params=None):
+        """Retrieves a live feed with the latest URLs submitted to VT.
+
+        Args:
+            resources: a dictionary with name and value for optional arguments
+        Returns:
+            A dict with the VT report.
+        """
+        params = params  or {}
```

Comment:
That's a python gotcha: mutable objects should never be passed as default argument! https://pythonconquerstheuniverse.wordpress.com/category/python-gotchas/